stream/popularity_mapper.js

```diff
@@ -0,0 +1,209 @@
+  historic: {
+    building: { _score: 5000 },
+    church: { _score: 5000 },
+    _score: 1000,
+  },
+  building: {
+    colour: { _score: 2000 },
+    material: { _score: 2000 },
+    supermarket: { _score: 2000 },
+    civic: { _score: 2000 },
+    government: { _score: 2000 },
+    hospital: { _score: 2000 },
+    train_station: { _score: 5000 },
+    transportation: { _score: 2000 },
+    university: { _score: 2000 },
+    public: { _score: 2000 },
+    sports_hall: { _score: 2000 },
+  },
+  height: { _score: 2000 },
+  start_date: { _score: 2000 },
+  opening_date: { _score: 2000 },
+
+  // properties found on tourist attractions
+  tourism: {
+    visitors: { _score: 2000 },
+    aquarium: { _score: 2000 },
+    attraction: { _score: 1000 },
+    museum: { _score: 2000 },
+    theme_park: { _score: 2000 },
+    zoo: { _score: 2000 },
+    _score: 1000
+  },
+  museum: { _score: 2000 },
+  museum_type: { _score: 2000 },
+  opening_hours: { _score: 1000 },
+  operator: { _score: 1000 },
+  fee: { _score: 2000 },
+
+  // closed and abandoned places
+  // https://wiki.openstreetmap.org/wiki/Key:disused:
+  disused: { _score: -100000 },
+  'disused:shop': { _score: -100000 },
+  'disused:amenity': { _score: -100000 },
+  'disused:railway': { _score: -100000 },
+  'disused:leisure': { _score: -100000 },
+  // https://wiki.openstreetmap.org/wiki/Key:abandoned:
+  abandoned: { _score: -100000 },
+  'abandoned:shop': { _score: -100000 },
+  'abandoned:amenity': { _score: -100000 },
+  'abandoned:railway': { _score: -100000 },
+  'abandoned:leisure': { _score: -100000 },
+
+  // types of public amenities
+  amenity: {
+    disused: { _score: -100000 },
+    college: { _score: 2000 },
+    library: { _score: 2000 },
+    school: { _score: 1000 },
+    university: { _score: 2000 },
+    bank: { _score: 1000 },
+    clinic: { _score: 1000 },
+    dentist: { _score: 1000 },
+    doctors: { _score: 1000 },
+    hospital: { _score: 2000 },
+    nursing_home: { _score: 1000 },
+    pharmacy: { _score: 1000 },
+    social_facility: { _score: 1000 },
+    veterinary: { _score: 1000 },
+    community_centre: { _score: 1000 },
+    music_venue: { _score: 1000 },
+    nightclub: { _score: 1000 },
+    planetarium: { _score: 1000 },
+    social_centre: { _score: 1000 },
+    theatre: { _score: 1000 },
+    courthouse: { _score: 1000 },
+    coworking_space: { _score: 1000 },
+    dojo: { _score: 1000 },
+    embassy: { _score: 1000 },
+    ferry_terminal: { _score: 2000 },
+    fire_station: { _score: 1000 },
+    marketplace: { _score: 1000 },
+    place_of_worship: { _score: 1000 },
+    police: { _score: 1000 },
+    post_office: { _score: 1000 },
+    prison: { _score: 1000 },
+    public_bath: { _score: 1000 },
+    townhall: { _score: 1000 }
+  },
+
+  // transportation
+  aerodrome: {
+    international: { _score: 10000 },
+    regional: { _score: 5000 }
+  },
+  iata: {
+    _score: 5000,
+    none: { _score: -5000 }
+  },
+  railway: {
```

Comment:
Looks like another related tag is [`public_transport=station`](https://wiki.openstreetmap.org/wiki/Tag:public%20transport=station?uselang=en-US):

>While the more common tag railway=station is used for all railway stations (i.e. including cargo), the public_transport=station tag is used only on the stations interesting for passenger transport.

This would hopefully help boost some more public transit stops like [Park & Market](https://www.openstreetmap.org/node/6184516702) which currently does not do very well in [our tests](https://github.com/pelias/acceptance-tests/blob/master/test_cases/search_poi.json#L94-L117)